src/datanode/src/greptimedb_telemetry.rs

```diff
@@ -0,0 +1,104 @@
+// Copyright 2023 Greptime Team
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use std::sync::Arc;
+
+use async_trait::async_trait;
+use common_greptimedb_telemetry::{
+    Collector, GreptimeDBTelemetry, GreptimeDBTelemetryTask, Mode as VersionReporterMode,
+    TELEMETRY_INTERVAL, TELEMETRY_UUID_KEY,
+};
+use object_store::{ErrorKind, ObjectStore};
+use servers::Mode;
+
+async fn get_uuid(file_name: &str, object_store: ObjectStore) -> Option<String> {
```

Comment:
problem: If users are using s3 as their object store, then we will write the uuid to it. Better to use `std::fs::File`. We need a path to store this uuid file. A good candidate is the `data_home` but it is not always provided as they might use S3 as object store.

Reply:
Now.In both standalone and distributed environments, the uuid is written to a fixed directory on the local disk.